dfdaemon/handler/transport.go

```diff
@@ -56,14 +56,17 @@ var compiler = regexp.MustCompile("^.+/blobs/sha256.*$")
 
 func needUseGetter(req *http.Request, location string) bool {
 	var useGetter bool
-	if req.Method == http.MethodGet {
-		if compiler.MatchString(req.URL.Path) {
-			return true
-		}
-		if location != "" {
-			return global.MatchDfPattern(location)
-		}
+	if req.Method != http.MethodGet {
+		return useGetter
```

Comment:
perhaps `return false` which makes it more clear.

Reply:
make sense